examples/migrating_trench/trench_hydro.py

```diff
@@ -0,0 +1,125 @@
+"""
```

Comment:
Add some comment that this is not the main example, just the setup that produces a spun up checkpoint...

Reply:
I have added a comment